### PR TITLE
Switch Serverless to pre-built files

### PR DESCRIPTION
**Pull request by aquarat**
This minor PR removes the serverless typescript plugin and makes Serverless use the `yarn build` output as an execution target.

There's also some extra detail in the deployment guide.

### CODE REVIEW

File: aws-lambda-deploy-guide.md [hunk 1]
```diff
@@ -2,14 +2,32 @@
...
+
+There are two main strategies for dealing with this:
+
+1. Deploy this Airseeker using a different service name in `serverless.yaml`, so that both Airseekers can run
+   concurrently. Wait for the new Airseeker to start. Once it does, ensure that you're satisfied that it is functioning
+   correctly (by referring to the target chains and/or CloudWatch logs). You may then revert the name of the service
+   stack to the old Airseeker and remove it or alternatively delete it using the AWS CloudFormation console.
+2. Deploy Airseeker using the default service name and start the new Airseeker immediately using the invoke commands
+   below.
+
```
Comment (amarthadan) [inline aws-lambda-deploy-guide.md]: I don't think point 1. has anything to do with Airseeker function not starting immediately. I guess what you're describing there is how would you "update" an Airseeker deployment?

**Review by amarthadan — APPROVED** [review]

File: aws-lambda-deploy-guide.md (hunk 1 above)
Comment (aquarat) [reply aws-lambda-deploy-guide.md]: ah yes, you're right. I find it's nice to run both the old and new Airseekers temporarily concurrently, check for correct functioning and then only remove the old Airseeker. It makes for a stable transition and gives one a chance to identify problems.

Comment (amarthadan) [reply aws-lambda-deploy-guide.md]: I agree with that but that should imo go into a different section.

Comment (acenolaza) [reply aws-lambda-deploy-guide.md]: Just wondering, could this cause any race condition when updating if both are using the same sponsor wallet?

**Comment by aquarat** [comment]
I forgot about this PR :disappointed: 
It has to be added to the pending release as the changes that were made to the `tsconfig.json` files broke the Serverless Typescript build and therefore Serverless must use the normal build output.
I'll merge this as-is. I don't believe the issues raised above are sufficiently problematic to block merging.

File: aws-lambda-deploy-guide.md (hunk 1 above)
Comment (aquarat) [reply aws-lambda-deploy-guide.md]: Better late than never: There would be a race condition, yes, but there is a series of race conditions every 14 minutes when the previous invocation and the new invocation overlap. The race conditions don't seem to be a problem (in terms of committed transactions) in practice.

**Pull request merged**